spec/lib/efile/id/id40_calculator_spec.rb

```diff
@@ -101,6 +101,57 @@
     end
   end
 
+  describe "Line 32a: Permanent Building Fund" do
+    context "has filing requirement, no blind filer, and no public assistance indicator" do
+      before do
+        intake.direct_file_data.total_income_amount = 40000
+        intake.direct_file_data.total_itemized_or_standard_deduction_amount = 2112
```

Comment:
ooh how come we didn't use the `no_filing_requirement` / `filing_requirement` `trait` in the factory for these?